samples/constitutions/default/actions.js

```diff
@@ -575,11 +575,11 @@ const actions = new Map([
     new Action(
       function (args) {
         checkType(
-          args.next_service_identity,
+          args.service_identity,
           "string",
-          "next service identity (PEM certificate)"
+          "service identity (PEM certificate)"
```

Comment:
I am not sure removing "next" is doing that much for clarity, it seemed nicely symmetrical with "previous"?